lib/workos/user_management.rb

```diff
@@ -481,19 +489,30 @@ def authenticate_user_with_code(code:, client_id:, ip_address: nil, user_agent:
           client_id: String,
           pending_authentication_token: String,
           authentication_challenge_id: String,
+          ip_address: T.nilable(String),
+          user_agent: T.nilable(String),
         ).returns(WorkOS::UserResponse)
       end
-      def authenticate_user_with_totp(code:, client_id:, pending_authentication_token:, authentication_challenge_id:)
+      def authenticate_with_totp(
+        code:,
+        client_id:,
+        pending_authentication_token:,
+        authentication_challenge_id:,
+        ip_address: nil,
+        user_agent: nil
+      )
         response = execute_request(
           request: post_request(
-            path: '/users/authenticate',
+            path: '/user_management/authenticate',
             body: {
               code: code,
               client_id: client_id,
               client_secret: WorkOS.config.key!,
               pending_authentication_token: pending_authentication_token,
+              grant_type: 'urn:workos:oauth:grant-type:email-verification:code',
```

Comment:
was this change intentional?

Reply:
Not sure if it's right? I followed the API reference: 

![CleanShot 2023-11-20 at 18 40 42](https://github.com/workos/workos-ruby/assets/48022589/114cc5a0-ccd5-438d-ac58-e4bff35d338b)

![CleanShot 2023-11-20 at 18 40 37](https://github.com/workos/workos-ruby/assets/48022589/6d809650-2bc3-456d-b882-dd4906271f65)

But it's confusing, it seems that we're missing a section there.

Reply:
yup there's a bug in the docs; `urn:workos:oauth:grant-type:mfa-totp` is the correct grant type